tests/pipeline/snapshots/snap_test_pipeline.py

```diff
@@ -10,7 +10,6 @@
 snapshots['TestPipeline.test_default_config test-pipeline'] = [
     {
         'app': {
-            'nameOverride': 'resources-custom-config-app1',
```

Comment:
Why are these removed? If we don't set anything streamsbootsrap will use the defined value in `value.yaml` see [here](https://github.com/bakdata/streams-bootstrap/blob/master/charts/streams-app/values.yaml#L1)

Shouldn't this be `full_name` if not set? according to your code above
```python
        if self.app.name_override is None:
            self.app.name_override = self.full_name
```

Reply:
it's only set in the [`to_helm_values`](https://github.com/bakdata/kpops/blob/1c4e3134b99de5ffb9964a4c9b16d7b70b620e83/kpops/components/base_components/helm_app.py#L161) method
it's invoked during `template`, `deploy`, etc.
Unless the user sets it in their pipeline definition, it's not part of `generate` output